### ISSUE TITLE
The Hay Group Singapore: The CEO and the new BMW

### ISSUE THREAD

**Issue by kadiennejamison** [kadiennejamison/The-Hay-Group-Singapore#1]
You’ve made it. You’re finally at the helm of this company or this Business Unit. Your new appointment crowns many years of hard work, and a bit of luck. You’ve read HBR articles in detail and a swarm of your favorite consultants are sending you congratulations notes.

In many ways, the situation is comparable to when you got delivery of your brand new BMW. It’s always a bit intimidating getting into a new car. The smell of leather and glue is nicely intoxicating but you worry about the controls. It would be silly to stall, or worse crash, as you take your first turn outside the dealership.

You know how to drive. You got your license eons ago, much like your forgotten MBA or engineering degree. You’ve driven thousands of miles on all types of roads and under many weather conditions. You have experience behind the wheel of many cars, from your rusty Honda at school to the convertible you bought when you cashed your first stock options.

It mirrors your career. You have been staff and line. You have headed three different countries. You’ve led teams in sales, marketing, IT and are considered an expert of the supply-chain. You have even restructured a couple of operations. Nothing fazes you.

The BMW sales guy shows you how the car controls work. Your short stint as the IT director in Canada becomes finally useful when he covers all the customization options, from gear-changing to suspension settings, from differentiated cabin climate to Bluetooth. Eventually you drive safely to your first board meeting which goes well.

You then organize a road trip with some cultural or sports alibi when in fact the plan is to see how your car really drives on that small winding road you so like. And as you discover the true performance envelope of your car, your grin grows. You understand how it corners, how you can accelerate and maintain traction, what its handling limits are. In essence you internalize the actual performance potential of your new car.

As your start your new job on Monday, your board reminds you that you are expected to deliver a deep corporate transformation leading to sustainably higher ROCE. Your <a href="http://blog.haygroup.com/the-ceo-and-the-new-bmw/">strategy consultants</a> tell you about the criticality of your first hundred days (make or break) and how they have a blueprint or a roadmap for you. Your CFO takes you through the details of your balance sheet and your cash position and she shows you a first draft of your communication to the analysts community. You are subjected to scores of presentations covering your market positions by geographies and business lines, your global manufacturing position, your patents and pending litigations. You are being shown the management systems at your disposal to run the company. But you get almost no information about your workforce. You don’t know its skills and how they will evolve. You have no visibility on its resilience. You have limited data on the turnover of the critical roles. You’re left to resort to your intuition.

In other words, you now know everything there is to know about the technical specifications of your company, but you don’t know how it will drive. This is why a Workforce Legacy analysis, which I will cover in my next blog, is as important as you <a href="http://blog.haygroup.com/">Strategic Roadmap</a>. Going somewhere with a car you don’t understand often won’t get you anywhere.
